app/helpers/users_helper.rb

```diff
@@ -5,4 +5,20 @@ def contribution_link(courses_user, text=nil, css_class=nil)
     options = { target: '_blank', class: css_class }
     link_to((text || courses_user.user.username), courses_user.contribution_url, options)
   end
+  def role(value)
```

Comment:
I suggest renaming this something more specific, like course_role_name. `value` should also have a more specific name, such as `courses_users_role`, to make it very clear how the method is supposed to be used.